src/coreclr/vm/eeconfig.h

```diff
@@ -656,6 +658,8 @@ class EEConfig
     bool fTieredPGO;
 #endif
 
+    bool fReadyToRun;
```

Comment:
This needs to be initialized, and the initialization should be de-duplicated with `ReadyToRunInfo::IsReadyToRunEnabled`.

Alternatively, you can delete this and just use the existing `ReadyToRunInfo::IsReadyToRunEnabled` method for the reporting purposes.

Reply:
Seems like it would make sense to either use `ReadyToRunInfo::IsReadyToRunEnabled()` everywhere, or to remove it and use `g_pConfig->ReadyToRun()` everywhere.